iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/setup/DeviceTwinCommon.java

```diff
@@ -41,12 +42,13 @@
  * Utility functions, setup and teardown for all device twin integration tests. This class should not contain any tests,
  * but any child class should.
  */
-public class DeviceTwinCommon extends MethodNameLoggingIntegrationTest
+public class DeviceTwinCommon extends IntegrationTest
 {
     // Max time to wait to see it on Hub
     protected static final long PERIODIC_WAIT_TIME_FOR_VERIFICATION = 100; // 0.1 sec
     protected static final long MAX_WAIT_TIME_FOR_VERIFICATION = 180000; // 180 sec
     protected static final long DELAY_BETWEEN_OPERATIONS = 200; // 0.2 sec
+    public static final long MULTITHREADED_WAIT_TIMEOUT_MS  = 5 * 60 * 1000; // 5 minutes
```

Comment:
eventually, I think we should consolidate all timeout into one central place for reusing...